source/agora/node/Ledger.d

```diff
@@ -538,7 +548,7 @@ version (unittest)
             super(config, params, new UTXOSet(":memory:"),
                 new MemBlockStorage(blocks),
                 new EnrollmentManager(":memory:", config.key_pair, params),
-                new TransactionPool(":memory:"));
+                new TransactionPool(":memory:"), (const ref Block) @safe {});
```

Comment:
This is the part I didn't see before. Thanks for making it clearer to me @TrustHenry.